Categories/UIView+TCAdditions.m

```diff
@@ -129,5 +129,19 @@ - (UIView *)tc_breadthFirstSubviewThatSatisfiesBlock:(BOOL (^)(UIView *))conditi
     return nil;
 }
 
+- (void)tc_autoLayoutPinToParentViewController {
```

Comment:
Well, multiple times in our projects, I've seen people setting one frame to the same size as another frame.  I think the point of this single method is that it seems to be something we're doing a lot, but being lazy about doing autolayout with it.  Most people do most of their autolayout in IB, so it's hard to see the point in adding an entire autolayout library to your pod file.

Definitely change the same (I don't know what I was thinking).  But for our internal projects, when I see someone setting frames equal, it's probably much easier to say "Hey, use this method from Thundercats, which is already in your pod file." then to say "Hey, write these 4 lines of code." or "Hey, go find an autolayout library."

Alternatively, we could pick an autolayout library we like and start including it in our projects by default?  But I'm not sure how much I like that idea.  I think most all of us are doing most of our autolayout work in IB.  And I don't think Apple's stuff is really that bad for the few times I have to use it.